tests/test_qpu.py

```diff
@@ -151,10 +156,16 @@ def circuit_Zmi():
         assert np.allclose(results[:3], 1.0, atol=2e-2)
         assert np.allclose(results[3:], -1.0, atol=2e-2)
 
+    @flaky(max_runs=10, min_passes=1)

```

Comment:
Since flaky should only be used when absolutely necessary, perhaps it's worth justifying in the docstring. Is it just because the number of shots here are small? 1 pass out of 10 runs doesn't seem like a high enough pass rate 😨 (I would imagine something like `max_runs=5`, `min_passes=3` being a lot more reasonable)

Reply:
This comment applies to all following uses of flaky

Reply:
Just realised this comment refers to a change that was made in #38, not here.

Reply:
Still adjusted them in there, thanks!